process.on("uncaughtException", (error) => {
  console.log("AAAAAAAAAA");
  console.error(error);
});
